etc/inc/disable-programs.inc

```diff
@@ -16,6 +16,7 @@ blacklist ${HOME}/.Natron
 blacklist ${HOME}/.PlayOnLinux
 blacklist ${HOME}/.PyCharm*
 blacklist ${HOME}/.Sayonara
+blacklist ${HOME}/Seafile/.seafile-data
```

Comment:
```
Run ./ci/check/profiles/sort-disable-programs.sh etc/inc/disable-programs.inc
sort: -:16: disorder: blacklist ${HOME}/.Steam
Error: Process completed with exit code 1.
```

Otherwise LGTM.